packages/itwinui-react/src/core/InputWithDecorations/InputWithDecorations.tsx

```diff
@@ -60,33 +66,40 @@ const InputWithDecorationsButton = React.forwardRef((props, ref) => {
   );
 
   return (
-    <IconButton
+    <InputFlexContainerButton
       ref={ref}
       size={size ?? contextSize}
-      styleType='borderless'
       disabled={localDisabled ?? isDisabled}
       {...rest}
     >
       {children}
-    </IconButton>
+    </InputFlexContainerButton>
   );
 }) as PolymorphicForwardRefComponent<
   'span',
```

Comment:
Unrelated to this PR: I think this should be "button" rather than "span"?

Reply:
Yes, fixed ([`7e93013` (#1881)](https://github.com/iTwin/iTwinUI/pull/1881/commits/7e93013d2504437a80d2a687bb5b274e721a1306)).